docker/README.md

```diff
@@ -156,7 +158,8 @@ There are three modes for plugins
 1. Plugins run in the `fiftyone-app` deployment
     - To enable this mode, use the file
       [./compose.plugins.yaml](./compose.plugins.yaml)
-      instead of [./compose.yaml](./compose.yaml)
+      instead of
+   [./compose.yaml](./compose.yaml)
```

Comment:
lolz. Now it is indentations are wrong : )

```suggestion
     [./compose.yaml](./compose.yaml)
```

Reply:
is fixed